### PR TITLE
Parseque: total parser combinators version 0.1.1 (support for Coq 8.17)

### PR DESCRIPTION
**Pull request by womeier**

### CODE REVIEW

File: released/packages/coq-parseque/coq-parseque.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "G. Allais"
+dev-repo: "git+https://github.com/gallais/parseque"
+bug-reports: "https://github.com/gallais/parseque/issues"
+homepage: "https://github.com/gallais/parseque"
+authors: ["G. Allais"]
+license: "GPL-3.0"
```
Comment (palmskog) [inline released/packages/coq-parseque/coq-parseque.0.1.1/opam:7]: Is this supposed to be `GPL-3.0-only` or `GPL-3.0-or-later`? Unfortunately, unqualified `GPL-3.0` is ambiguous. (cc: @gallais)

Comment (gallais) [reply released/packages/coq-parseque/coq-parseque.0.1.1/opam:7]: I'm happy with `-or-later`.

Comment (palmskog) [inline released/packages/coq-parseque/coq-parseque.0.1.1/opam:7]:
```suggestion
license: "GPL-3.0-or-later"
```

**Comment by womeier** [comment]
thanks!

**Pull request merged**